### ISSUE TITLE
Migration issue updating 1.1.0 => 1.1.1

### ISSUE THREAD

**Issue by ccchapman** [mmikkel/IncognitoField-Craft3#5]
**Steps to Reproduce:**

1. Have Incognito Field v1.1.0 installed on Craft CMS 3
2. Update with `composer update`
3. Visit CP and receive message for migrations
4. Receive issue with `mmikkel\incognitofield\migrations\m190226_225231_craft3`

```
Error: Class 'craft\db\Table' not found

Migration: mmikkel\incognitofield\migrations\m190226_225231_craft3

Output:

Exception: Class 'craft\db\Table' not found (/home/vagrant/project/vendor/mmikkel/incognito-field/src/migrations/m190226_225231_craft3.php:36)
#0 /home/vagrant/project/vendor/mmikkel/incognito-field/src/migrations/m190226_225231_craft3.php(24): mmikkel\incognitofield\migrations\m190226_225231_craft3->migratePlainTextSettings()
#1 /home/vagrant/project/vendor/craftcms/cms/src/db/Migration.php(56): mmikkel\incognitofield\migrations\m190226_225231_craft3->safeUp()
#2 /home/vagrant/project/vendor/craftcms/cms/src/db/MigrationManager.php(243): craft\db\Migration->up(true)
#3 /home/vagrant/project/vendor/craftcms/cms/src/db/MigrationManager.php(163): craft\db\MigrationManager->migrateUp(Object(mmikkel\incognitofield\migrations\m190226_225231_craft3))
#4 /home/vagrant/project/vendor/craftcms/cms/src/services/Updates.php(215): craft\db\MigrationManager->up()
#5 /home/vagrant/project/vendor/craftcms/cms/src/controllers/BaseUpdaterController.php(519): craft\services\Updates->runMigrations(Array)
#6 /home/vagrant/project/vendor/craftcms/cms/src/controllers/UpdaterController.php(208): craft\controllers\BaseUpdaterController->runMigrations(Array, 'restore-db')
#7 [internal function]: craft\controllers\UpdaterController->actionMigrate()
#8 /home/vagrant/project/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#9 /home/vagrant/project/vendor/yiisoft/yii2/base/Controller.php(157): yii\base\InlineAction->runWithParams(Array)
#10 /home/vagrant/project/vendor/craftcms/cms/src/web/Controller.php(104): yii\base\Controller->runAction('migrate', Array)
#11 /home/vagrant/project/vendor/yiisoft/yii2/base/Module.php(528): craft\web\Controller->runAction('migrate', Array)
#12 /home/vagrant/project/vendor/craftcms/cms/src/web/Application.php(284): yii\base\Module->runAction('updater/migrate', Array)
#13 /home/vagrant/project/vendor/craftcms/cms/src/web/Application.php(670): craft\web\Application->runAction('updater/migrate')
#14 /home/vagrant/project/vendor/craftcms/cms/src/web/Application.php(238): craft\web\Application->_processUpdateLogic(Object(craft\web\Request))
#15 /home/vagrant/project/vendor/yiisoft/yii2/base/Application.php(386): craft\web\Application->handleRequest(Object(craft\web\Request))
#16 /home/vagrant/project/web/index.php(21): yii\base\Application->run()
#17 {main}
```

**Other Details:**
Craft CMS: 3.0.41

I [expect this migration will run successfully on Craft CMS 3.1](https://docs.craftcms.com/api/v3/craft-db-table.html). If this is a requirement for 1.1.1 of Incognito Field, then the Composer requirement should be updated accordingly. (Or the migration should only run/support upgrading to 3.1 and the plugin is otherwise still usable?)

cc: @mmikkel @carlcs

**Comment by carlcs** [comment]
Sorry @ccchapman, I just created a PR that fixes the issue. #6

**Comment by mmikkel** [comment]
Hey @ccchapman, thanks for reporting!

I've just merged #6 and published a new release [v. 1.1.1.1](https://github.com/mmikkel/IncognitoField-Craft3/releases/tag/1.1.1.1). Thanks @carlcs!

**Comment by ccchapman** [comment]
@carlcs @mmikkel Thanks for the quick update. I can confirm that 1.1.1.1 resolves my issue.
